ui/src/ScholarReader.tsx

```diff
@@ -412,28 +413,31 @@ class ScholarReader extends React.PureComponent<ScholarReaderProps, State> {
   }
 
   scrollSymbolIntoView() {
-    setTimeout(() => {
-      const { selectedSymbol, pdfViewer, pages } = this.state;
-      if (pdfViewer && selectedSymbol) {
-        const symBounds = selectedSymbol.bounding_boxes[0];
-        const { left } = selectors.divDimensionStyles(
-          pages[symBounds.page + 1].view, symBounds
-        );
-        
-        // Left = left position on the pdf page, scroll left = left offset of the pdf
-        // 200 = extra padding from the pdf side panel. 
-        const relativeSymbolLeftPosition = left - pdfViewer.container.scrollLeft + 200;
-        // window.innerWidth = possible visible area of the viewport, 
-        // 470 = width now obscured by the open drawer. 
-        const viewableViewportWidth = window.innerWidth - 470;
-        console.log(`Left Position: ${relativeSymbolLeftPosition} > ${viewableViewportWidth} | moving: ${relativeSymbolLeftPosition - viewableViewportWidth}`);
-        if (relativeSymbolLeftPosition > viewableViewportWidth) {
-          // Here we are moving the pdf to the left by the amount the symbol was obscured by
-          // + some padding to make it a little bit away from the drawer. 
-          pdfViewer.container.scrollLeft += Math.max((relativeSymbolLeftPosition - viewableViewportWidth) + 50, 0);
-        }
-      } 
-    }, 0);
+    const { selectedSymbol, pdfViewer, pages } = this.state;
+    if (pdfViewer && selectedSymbol) {
+      const symBounds = selectedSymbol.bounding_boxes[0];
+      const { left } = selectors.divDimensionStyles(
+        pages[symBounds.page + 1].view, symBounds
```

Comment:
It might be a good idea to check that `pages[symBounds.page + 1]` is defined before using its value here